tests/test_package/test_convert.py

```diff
@@ -0,0 +1,16 @@
+from tidy3d.web.cli import tidy3d_cli
+from tidy3d.web.cli.converter import converter_arg
+from pathlib import Path
+
+
+def test_tidy3d_converter():
+    lsf_path = str(Path.cwd()) + "/tests/data/example.lsf"
+    converter_arg(lsf_path, "lsf_converter_test.py")
+    file_path = Path("lsf_converter_test.py")
+    assert file_path.exists()
+    Path.unlink(file_path)
```

Comment:
note, using `tmp_path`, should be no need to do this, we can also check that the out file exists through 
```
os.path.exists(path)
```
if `path` is just a string.